### DISCUSSION TITLE
Starting xkill (via a keyboard shortcut) doesn't seems to work

### DISCUSSION THREAD

**Discussion by brunetton** [i3/i3#4533] [Q&A] [answered]
Hi, 

I'm trying to assign a shortcut to `xkill` command; so I added to my i3 config file:

```
bindsym $mod+Shift+x exec --no-startup-id xkill
```

But this doesn't do anything (I restarted i3, of course); ie when I press the sortcut this doesn't do anything.

- This
    ```
    bindsym $mod+Shift+x exec --no-startup-id notify-send "Hello word"
    ```
    works as excepted (I just replaced xkill command by notify-send "Hello word"). So this is not a problem of unavailable shortcut
- I checked that `xkill` is available in path (I also tried with full path `/usr/bin/xkill` in config file)
- I restarted i3 with logs and had a look in the output of `i3-dump-log -f` command and I can't see anything special:
   ```
    22/09/2021 15:40:08 - commands_parser.c:parse_command:266 - COMMAND: *exec --no-startup-id xkill*
    22/09/2021 15:40:08 - commands.c:cmd_criteria_init:153 - Initializing criteria, current_match = 0x55654dd44380
    22/09/2021 15:40:08 - commands.c:cmd_exec:1210 - should execute xkill, no_startup_id = 1
    22/09/2021 15:40:08 - executing: xkill
    22/09/2021 15:40:08 - commands.c:cmd_criteria_init:153 - Initializing criteria, current_match = 0x55654dd44380
    22/09/2021 15:40:08 - ipc.c:ipc_send_binding_event:1627 - Issue IPC binding run event (sym = x, code = 0)
    22/09/2021 15:40:08 - handlers.c:handle_event:1357 - event type 3, xkb_base 85
    22/09/2021 15:40:08 - key_press.c:handle_key_press:23 - KeyRelease 53, state raw = 0x41
    22/09/2021 15:40:08 - bindings.c:get_binding_from_xcb_event:316 - (removed capslock, state = 0x41)
    22/09/2021 15:40:08 - bindings.c:get_binding_from_xcb_event:341 - (transformed keyboard group, state = 0x10041)
    22/09/2021 15:40:08 - bindings.c:get_binding:238 - binding_keycode->modifiers = 69, modifiers_mask = 69, modifiers_state = 65, mods_match = no
   ```
    The "should execute xkill" makes me think that my config file should be okay

Any tips here ?
Should I open an issue ?

Thanks !

**Comment by Airblader** [comment, marked as answer]
This is the same as #4504, please try the solution there (adding `--release`).

**Reply by brunetton** [reply]
Thanks @Airblader ! This worked 🎉🎉🎉

So in my case, this fixed the problem:

```
bindsym --release $mod+Shift+x exec --no-startup-id xkill
```

**Comment by orestisfl** [comment]
See --release in the user guide. I.e. exec --release

**Reply by brunetton** [reply]
Thanks @orestisfl. This is the good idea but the syntax is `bindsym --release [...] exec [..]`

**Reply by orestisfl** [reply]
Whoops, my bad
